docs/license.rst

```diff
@@ -9,4 +9,4 @@ Ccdproc License
 
 Ccdproc is licensed under a 3-clause BSD style license:
 
-.. include:: ../licenses/LICENSE.rst
+.. include:: ../LICENSE.rst
```

Comment:
I leave you to decide on this for ccdproc, astropy core does exactly the same as above.